package.json

```diff
@@ -59,7 +61,7 @@
     "typescript": "^5.8.3"
   },
   "peerDependencies": {
-    "@map-colonies/js-logger": "^1.0.1",
-    "@map-colonies/mc-utils": "^3.1.0"
+    "@map-colonies/js-logger": "^3.0.1",
```

Comment:
Latest is [v3.0.2](https://github.com/MapColonies/js-logger/releases/tag/v3.0.2)

Reply:
is not

Reply:
please check again